### ISSUE TITLE
Parsing error when using `code-block` directive

### ISSUE THREAD

**Issue by jrbourbeau** [keewis/blackdoc#150]
When using `blackdoc` (v0.3.7) on this `.rst` snippet 

```rst

.. code-block:: python

   >>> import dask.dataframe as dd
``` 

I get 

```
❯ python -m blackdoc test.rst
error: cannot format /Users/james/projects/dask/dask/test.rst: Cannot parse: 4:0: >>> import dask.dataframe as dd
Oh no! 💥 💔 💥
1 file fails to reformat.
```

However, when I update the `.rst` to 

```rst

.. ::

   >>> import dask.dataframe as dd
```

which should be equivalent, `blackdoc` is happy.

**Comment by keewis** [comment]
thanks for the report, @jrbourbeau. This should be pretty easy to fix (there's similar code for the `ipython` prompt), but it might take me some time to create a PR.

**Comment by keewis** [comment]
Note that
```rst
.. ::

    >>> import dask.dataframe as dd
```
is equivalent to
```rst
>>> import dask.dataframe as dd
```
(i.e. `blackdoc` does not care about the block for doctest lines... except the bug reported here, of course)

However, this:
```rst
.. ::

    import dask.dataframe as dd
```
or
```rst
description::

    import dask.dataframe as dd
```
will most likely always be ignored, because the language depends on a `sphinx` setting / a directive somewhere earlier in the document.

In any case, the bug should be fixed by #151.

**Comment by jrbourbeau** [comment]
Thanks for the fix @keewis! Looking forward to taking this for a spin
